config/lib/i18n.js

```diff
@@ -33,7 +37,7 @@ i18n
   .use(backend)
   .use(reactI18nextModule) // passes i18n down to react-i18next
   .init({
-    lng: 'eng',
+    lng: languages[0].code,
     // allow keys to be phrases having `:`, `.`
     nsSeparator: false,
     keySeparator: false,
```

Comment:
Note duplicate `keySeparator`, fixed in https://github.com/Trustroots/trustroots/commit/39037f3f58912af8ddbdf9e77cd31342d0a10353

Reply:
rebased